tasks/framework-tools/src/frameworkSyncToProject.mjs

```diff
@@ -91,11 +92,17 @@ chokidar
 
     console.log()
     logStatus(`Building ${packageName}...`)
-    buildPackages([packageJsonPath])
+    try {
+      buildPackages([packageJsonPath], { clean: true })
 
-    console.log()
-    logStatus(`Copying ${packageName}...`)
-    copyFrameworkFilesToProject(projectPath, [packageJsonPath])
+      console.log()
+      logStatus(`Copying ${packageName}...`)
+      copyFrameworkFilesToProject(projectPath, [packageJsonPath])
```

Comment:
If there's an error building a package, it doesn't make sense to proceed in copying it over so I included it in the try-catch block